_includes/responsive_video

```diff
@@ -0,0 +1,11 @@
+{% capture video_id %}{{ include.id }}{% endcapture %}
+{% capture video_provider %}{{ include.provider }}{% endcapture %}
+
+<!-- Courtesy of embedresponsively.com //-->
+<div class="responsive-video-container">
+{% if video_provider == "vimeo" %}
+  <iframe src="http://player.vimeo.com/video/{{ video_id }}" frameborder="0" webkitAllowFullScreen mozallowfullscreen allowFullScreen></iframe>
```

Comment:
Maybe just do `https://`, I just checked the embeds on Vimeo and that's what they default to. The YouTube one is that way as well.